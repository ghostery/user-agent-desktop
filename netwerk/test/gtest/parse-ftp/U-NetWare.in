ftp> open netlab2.usu.edu
Connected to netlab2.usu.edu.
220 Service Ready for new User
Name (netlab2.usu.edu:cyp): 
331 Enter E-Mail id as Password
230 User anonymous Logged in Successfully
Remote system type is NETWARE.
ftp> syst
215 NETWARE  Type : L8
ftp> ls
200 PORT Command OK
150 Opening data connection
total 0
- [RWCEAFMS] NFAUUser                          192 Apr 27 15:21 HEADER.html
d [RWCEAFMS] jrd                               512 Jul 11 03:01 allupdates
d [RWCEAFMS] jrd                               512 Dec 05  2001 apps
d [RWCEAFMS] jrd                               512 Oct 21  2001 bsuk00
d [RWCEAFMS] jrd                               512 Oct 21  2001 bsuk2001
d [RWCEAFMS] jrd                               512 Oct 21  2001 bsuk99
d [RWCEAFMS] jrd                               512 Oct 21  2001 bsus2000
d [RWCEAFMS] jrd                               512 Oct 21  2001 bsus2001
d [RWCEAFMS] jrd                               512 Mar 27 20:23 bsus2002
d [RWCEAFMS] jrd                               512 Oct 21  2001 bsus99
d [RWCEAFMS] jrd                               512 Oct 21  2001 drivers
d [RWCEAFMS] jrd                               512 Oct 24  2001 freebird
d [RWCEAFMS] jrd                               512 Apr 30 13:56 kermit
d [RWCEAFMS] jrd                               512 Oct 21  2001 macipx
d [RWCEAFMS] jrd                               512 Jun 13 01:45 misc
d [RWCEAFMS] jrd                               512 Oct 21  2001 netwatch
d [RWCEAFMS] jrd                               512 Oct 21  2001 netwire
d [RWCEAFMS] jrd                               512 Oct 21  2001 odi
d [RWCEAFMS] jrd                               512 Oct 21  2001 pktdrvr
d [RWCEAFMS] jrd                               512 Mar 02 20:23 rfc
d [RWCEAFMS] jrd                               512 Oct 30  2001 UnixWare
d [RWCEAFMS] jrd                               512 Oct 21  2001 updates
- [RWCEAFMS] jrd                            666258 Oct 30  1992 ENCAPS10.PS
- [RWCEAFMS] jrd                             25085 Oct 28  1992 ENCAPS10.TXT
- [RWCEAFMS] jrd                            611989 Jan 13  1993 ENCAPS11.PS
- [RWCEAFMS] jrd                             27134 Jan 12  1993 ENCAPS11.TXT
- [RWCEAFMS] jrd                               372 Jan 13  1993 README.TXT
226 Transfer Complete
ftp> site help
214-Only DOS and LONG site commands are supported for Anonymous Users 
 DOS : 
 	The DOS command shows the file/directory permissions in DOS format. 
 LONG	: 
 	The LONG command shows the file/directory permissions in LONG format. 
214 End of Help
ftp> close
221 Closing Session
ftp> 