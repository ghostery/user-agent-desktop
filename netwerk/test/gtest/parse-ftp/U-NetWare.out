<!-- This Source Code Form is subject to the terms of the Mozilla Public
   - License, v. 2.0. If a copy of the MPL was not distributed with this
   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->

04-27-2002  15:21:00                  192 HEADER.html
07-11-2002  03:01:00       <DIR>          allupdates
12-05-2001  00:00:00       <DIR>          apps
10-21-2001  00:00:00       <DIR>          bsuk00
10-21-2001  00:00:00       <DIR>          bsuk2001
10-21-2001  00:00:00       <DIR>          bsuk99
10-21-2001  00:00:00       <DIR>          bsus2000
10-21-2001  00:00:00       <DIR>          bsus2001
03-27-2002  20:23:00       <DIR>          bsus2002
10-21-2001  00:00:00       <DIR>          bsus99
10-21-2001  00:00:00       <DIR>          drivers
10-24-2001  00:00:00       <DIR>          freebird
04-30-2002  13:56:00       <DIR>          kermit
10-21-2001  00:00:00       <DIR>          macipx
06-13-2002  01:45:00       <DIR>          misc
10-21-2001  00:00:00       <DIR>          netwatch
10-21-2001  00:00:00       <DIR>          netwire
10-21-2001  00:00:00       <DIR>          odi
10-21-2001  00:00:00       <DIR>          pktdrvr
03-02-2002  20:23:00       <DIR>          rfc
10-30-2001  00:00:00       <DIR>          UnixWare
10-21-2001  00:00:00       <DIR>          updates
10-30-1992  00:00:00               666258 ENCAPS10.PS
10-28-1992  00:00:00                25085 ENCAPS10.TXT
01-13-1993  00:00:00               611989 ENCAPS11.PS
01-12-1993  00:00:00                27134 ENCAPS11.TXT
01-13-1993  00:00:00                  372 README.TXT
